---
title: Daniel B. Goodman
layout: welcome
---
### Using novel high-throughput approaches to understand and engineer the immune system.

Our understanding of how genomes control biological function is limited by our ability to make modifications and test hypotheses. As it becomes cheaper and faster to both [chemically synthesize DNA](http://www.nature.com/nmeth/journal/v11/n5/abs/nmeth.2918.html) and to [read from cells with next-generation sequencing](https://www.nature.com/articles/nbt.2421), we can identify relationships between DNA sequences and cellular outcomes at an unprecedented scale. This large amount of high-dimensional data can combined with advancements in artificial intelligence that can both rapidly predict phenotypes and generate new sequences to test.

During my PhD in George Church's lab at Harvard Medical School, I studied interactions between [bacterial transcription and translation](https://www.ncbi.nlm.nih.gov/pubmed/24072823), built and measured libraries of tuneable synthetic biosensors, and constructed [a new verison of the E. coli genome](https://www.ncbi.nlm.nih.gov/pubmed/24136966) capable of incorporating new synthetic amino acids into its proteins. I also built a high-throughput microbial genome design and analysis software platform called [Millstone](https://churchlab.github.io/millstone/).

As a [Jane Coffin Childs Postdoctoral Fellow](http://www.jccfund.org/) at UCSF, I'm currently applying these high-throughput synthetic approaches to [engineer T cells](https://www.nature.com/nature/journal/v545/n7655/full/nature22395.html) for the treatment of cancer and autoimmune disease. I'm working in the [Bluestone](https://bluestonelab.ucsf.edu/people), [Roybal](https://www.roybal-lab.org/people/), and [Marson](http://marsonlab.ucsf.edu/) labs.